src/main/java/com/google/devtools/build/lib/starlarkbuildapi/test/TestEnvironmentInfoApi.java

```diff
@@ -28,4 +29,10 @@
       doc = "A dict containing environment variables which should be set on the test action.",
       structField = true)
   Map<String, String> getEnvironment();
+
+  @StarlarkMethod(
+      name = "inherited_environment",
+      doc = "A list of variables that the test action should inherit from the environment.",
```

Comment:
Can you make this more precise? From which "environment" are they inherited?

Reply:
I added "shell" to clarify this.